Styling/schema/style.meta.schema.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema" : "http://json-schema.org/draft-03/schema",
+    "title" : "meta",
+    "type" : "object",
+    "description" : "A series of property names and the expression to evaluate for the value of each property.",
+    "properties" : {
+    },
+    "additionalProperties" : {
+        "$ref" : "style.metaProperty.schema.json"
```

Comment:
I said "A valid 3D Tiles style expression", should I list every possible type as a possibility? Because it could also be a color object or a regex.